deploy.sh

```
@@ -2,14 +2,27 @@
+ENV_FILE=".env"
 
 # 현재 시간을 로그에 기록
 echo "===== 배포 시작: $(date) =====" >> $LOG_FILE
 
+# 환경 변수 파일 확인
+if [ -f "$ENV_FILE" ]; then
+  echo "환경 변수 파일(.env)이 있습니다. 애플리케이션에 환경 변수를 제공합니다." >> $LOG_FILE
+  
+  # 환경 변수 로드
+  set -a
+  source $ENV_FILE
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)